tools/test.py

```diff
@@ -195,7 +195,10 @@ def main():
         if args.eval:
             eval_kwargs = cfg.get('evaluation', {}).copy()
             # hard-code way to remove EvalHook args
-            for key in ['interval', 'tmpdir', 'start', 'gpu_collect']:
+            for key in [
```

Comment:
Other scripts such as `benchmark.py` may also be affected.

Reply:
`benchmark.py` doesn't call anything with `cfg.evaluation`, so it should be fine?

Reply:
Oh sorry, what I mean is the script `eval_metric.py` .